libs/sdk-backend-tiger/src/backend/workspace/execution/executionResult.ts

```diff
@@ -67,7 +67,12 @@ export class TigerExecutionResult implements IExecutionResult {
 
     public async readAll(): Promise<IDataView> {
         const executionResultPromise = this.authCall((client) =>
-            client.execution.executionResult(this.workspace, this.resultId),
+            client.executionResult
+                .retrieveResult({
+                    workspaceId: this.workspace,
+                    resultId: this.resultId,
+                })
+                .then(({ data }) => data),
```

Comment:
not related to this PR but we rather double-check whether the call without paging actually returns all the data or some first-default-page only. if the latter is the case we have a bug.

Reply:
hm, I'll try to check it in AD, good point